### PR TITLE
Some more fixes, it's unitystation bug hunting month apparently

### PR DESCRIPTION
**Pull request by MaxIsJoe**
CL: [Fix] Players who do not have any character sheets will be prevented from going to the "Select Job" screen.
CL: [Fix] Fixed the mining scanner having an incorrect size.

### CODE REVIEW

File: UnityProject/Assets/Scripts/UI/Systems/PreRound/GUI_PreRoundWindow.cs [hunk 1]
```diff
@@ -336,6 +344,10 @@ public void CloseMapLoadingPanel()
 			normalWindows.SetActive(true);
 			mapLoadingPanel.SetActive(false);
 			UpdateLoadingBar("Preparing..", 0.1f);
+			if (PlayerManager.CharacterManager.Characters.Any() == false)
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/UI/Systems/PreRound/GUI_PreRoundWindow.cs:347]: this also could include the countdown?

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/UI/Systems/PreRound/GUI_PreRoundWindow.cs:347]: wdym?

Comment (Bod9001) [reply UnityProject/Assets/Scripts/UI/Systems/PreRound/GUI_PreRoundWindow.cs:347]: what happens if you don't have a character settings and you click ready? during the countdown

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/UI/Systems/PreRound/GUI_PreRoundWindow.cs:347]: It just activates the root window

Comment (Bod9001) [reply UnityProject/Assets/Scripts/UI/Systems/PreRound/GUI_PreRoundWindow.cs:347]: ok cool

**Review by Bod9001 — APPROVED** [review]

**Pull request merged**